agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -500,13 +500,37 @@ void backhaul_manager::after_select(bool timeout)
         }
     }
 
+    // Send ap metrics query every reporting_interval_s seconds
+    static std::chrono::steady_clock::time_point ap_metric_interval =
```

Comment:
There was a placeholder for this code in backhaul_fsm_main(), inside case EState::OPERATIONAL

Reply:
Fixed